app/services/enqueue_index_jobs.rb

```diff
@@ -0,0 +1,70 @@
+# Indexes a book version thru the SQS queuing system
+#
+# @params indexing_version e.g., Search::BookVersions::I1::IndexingStrategy::VERSION
+class EnqueueIndexJobs
+  def initialize
+    @todo_jobs_queue = TodoJobsQueue.new
+  end
+
+  def process_rex_releases
```

Comment:
changed name to `execute` in the PR.  B/c does an outside caller really care that this starts by pulling rex releases?  I think this Ruby PORO is still just a script but now one that has attributes/state.  So I like just saying `do_it` or `call` or similar.

Reply:
this is just my opinion...so would love to discuss more...but i've disliked using "meaningless" method names like execute or do_it or call (in services).  it adds nothing to the code documentation and understanding, when what is it "calling" or "doing' is adding to easier code understanding.  method names are the source for documenting what the code or method is doing, hopefully a single responsibility.

Reply:
this method is what could be the inlined body of the rake script, so to me the name of this method is really "enqueue_index_jobs".  of course that is also the class name here, which is why a "do_it" method makes sense to me (because `EnqueueIndexJobs.new.enqueue_index_jobs` is repetitive).  Said another way, `EnqueueIndexJobs` is a glorified function (a function that is an object so that it can have state), and functions are `call`ed.  Does `process_rex_releases` help here with the understanding of what's going on?  If a new user came to this code, would it be obvious that `process_rex_releases` is what should be called to run this "enqueue index jobs" function?  To me, it isn't really obvious other than it being the only public method.

Reply:
thanks @jpslav.  We argued over this for hours at my previous job.... i'm cool with this and agree...that the idea of the service object is that of a glorified function object, to invoke call or execute on. 

i do hope tho that we dont pack our (production) rake scripts with functional code.  same reasoning as packing a controller with functional code (harder to test, less SRP, etc).